services/blockchain-indexer/shared/indexer/transactionProcessor/index.js

```diff
@@ -37,21 +37,21 @@ const getCommandProcessors = async (moduleName) => requireAll({
 
 const buildModuleCommandProcessorMap = async () => {
```

Comment:
This will adjustment according to the requested changes within the module-specific implementations of the transactionProcessor.